src/lib/components/forms/Input.svelte

```diff
@@ -14,7 +14,7 @@
 <div class="group" class:onError={error}>
 	{#if !noLabel}
 		<label for={name}>
-			<slot>Label*</slot>
```

Comment:
Required fields should be specified via HTML (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/required) and not hard-coded as string.